plugins/jira-dashboard-backend/src/service/standaloneServer.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  HostDiscovery,
+  ServerTokenManager,
+  createServiceBuilder,
+  loadBackendConfig,
+} from '@backstage/backend-common';
+import { Server } from 'http';
+import { Logger } from 'winston';
+import { createRouter } from './router';
+import { IdentityApi } from '@backstage/plugin-auth-node';
+
+export interface ServerOptions {
+  port: number;
+  enableCors: boolean;
+  logger: Logger;
+}
+
+export async function startStandaloneServer(
+  options: ServerOptions,
+): Promise<Server> {
+  const logger = options.logger.child({ service: 'jira-dashboard-backend' });
+  const config = await loadBackendConfig({ logger, argv: process.argv });
+  const tokenManager = ServerTokenManager.noop();
+  logger.debug('Starting application server...');
+  const router = await createRouter({
+    logger,
+    config: config,
+    discovery: HostDiscovery.fromConfig(config),
+    identity: {} as IdentityApi,
+    tokenManager,
+  });
+
+  let service = createServiceBuilder(module)
+    .setPort(options.port)
+    .addRouter('/jira-dashboard', router);
+  if (options.enableCors) {
+    service = service.enableCors({ origin: 'http://localhost:3000' });
```

Comment:
Can this be passed as a an env var as there is flexibility for developers to change port

Reply:
This just the default that comes with the skeleton, basically every plugin in the repo does exactly this - i'd be happy to allow it for now and iterate later, if the rest of the pr looks good